PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1821,24 +1824,34 @@ void FindReplaceDlg::findAllIn(InWhat op)
 		cmdid = WM_FINDINFILES;
 	else if (op == CURRENT_DOC)
 		cmdid = WM_FINDALL_INCURRENTDOC;
-
-	if (!cmdid) return;
+	else
+		return;
 
 	if (::SendMessage(_hParent, cmdid, 0, 0))
 	{
-		if(_findAllResult == 1)
-			wsprintf(_findAllResultStr, TEXT("1 hit"));
-		else
-			wsprintf(_findAllResultStr, TEXT("%d hits"), _findAllResult);
-		if (_findAllResult) 
+		if (_findAllResult > 0) 
 		{
+			if (_findAllResult == 1)
+				wsprintf(_findAllResultStr, TEXT("1 hit"));

```

Comment:
> I'll do it.

I have a feeling that if you do it, it's more likely to get merged than if I were to do it :)